tests/memoryview/numpy_memoryview.pyx

```diff
@@ -216,7 +202,6 @@ def test_transpose_type(a):
     print m_transpose[6, 4]
 
 
-@testcase_numpy_1_5
 def test_numpy_like_attributes(cyarray):
     """
     For some reason this fails in numpy 1.4, with shape () and strides (40, 8)
```

Comment:
This comment can be removed as well...

Reply:
Right - I've removed that as well. I think I'll try to stop digging up Numpy archaeology at this stage, since it's slightly distracting from what I really wanted to do (stop PyPy running tests that are incompatible with its GC)

Reply:
Yeah, sorry for the distraction... 😅 

In any case, having a green CI again is worth celebrating! 🥳